pkg/gui/controllers/local_commits_controller.go

```diff
@@ -759,3 +764,18 @@ func (self *LocalCommitsController) paste() error {
 func (self *LocalCommitsController) isHeadCommit() bool {
 	return models.IsHeadCommit(self.model.Commits, self.context().GetSelectedLineIdx())
 }
+
+func isChangeOfRebaseTodoAllowed(action todo.TodoCommand) bool {
+	allowedActions := set.NewFromSlice(
```

Comment:
no need to use a set if we're only going to use it once. I'd just use `lo.Contains` on the slice. It is indeed the case that we have a few too many of these slices/sets util packages. I prefer using `lo` when I have the choice: it's more comprehensive than my own generics packages

Reply:
Good to know. See 518ce4192fd3.